rllib/tests/backward_compat/old_ppo.py

```diff
@@ -0,0 +1,32 @@
+from ray.rllib.agents import with_common_config
```

Comment:
Why name this old_ppo.py?

Reply:
Well, it's the old PPO python config dict.